2020/10/2020-10-23-RIAA.md

```diff
@@ -4,115 +4,27 @@ GitHub
-including enforcing their copyrights and common law rights on the Internet.
+We here at the RIIA are complete and utter idiots, and we have no idea how anything works.
 
-Copyright Violations. We have learned that your service is hosting the youtube-dl source code
-on its network at the following locations, among others:
+As we have a monopoly on 85% of the music market in America, we've basically got you fucks by the balls, and you can't do jack shit about it, because we have our artists by the balls too.
 
-https://github.com/ytdl-org/youtube-dl  
-https://github.com/ytdl-org/youtube-dl/tree/gh-pages  
-http://ytdl-org.github.io/youtube-dl/  
-https://github.com/benkeung/youtube-dl  
-https://github.com/cyberjacob/youtube-dl  
-https://github.com/elaopinska/youtube-dl  
-[private]    
-https://github.com/huangciyin/youtube-dl  
-https://github.com/jckelley/youtube-dl  
-https://github.com/LouisPlisso/youtube-dl  
-https://github.com/ojauch/youtube-dl  
-https://github.com/rbrito/youtube-dl  
-https://github.com/successLee/youtube-dl  
-https://github.com/trammel/youtube-dl  
-https://github.com/vs9390/youtube-dl  
-https://github.com/zackfern/youtube-dl  
-https://github.com/tosuch/youtube-dl  
-https://github.com/pornophage/youtube-dl  
-https://github.com/tejaskhot/youtube-dl  
-https://github.com/VideoUtils/youtube-dl  
-https://github.com/798221028/youtube-dl  
+Since we all know American law applies to the entire world, we can do whatever the fuck we want lol
 
-The above list includes a representative sample of the youtube-dl forks of the youtube-dl
-source code being hosted on GitHub. Based on our review of the representative sample noted
-above, we have a good faith belief that most of the youtube-dl forks are infringing to the same
-extent as the parent repository.
+Most of this comes in the form of taking down anyone we don't like! It's great!
 
-The clear purpose of this source code is to (i) circumvent the technological protection measures
-used by authorized streaming services such as YouTube, and (ii) reproduce and distribute music
-videos and sound recordings owned by our member companies without authorization for such
-use. We note that the source code is described on GitHub as “a command-line program to
-download videos from YouTube.com and a few more sites.”1
+Because we're such morons, we believe that youtube-dl is infringing on our copyright with code we don't own, and again, because we've got you by the balls, you need to take it down.
 
-We also note that the source
-code prominently includes as sample uses of the source code the downloading of copies of our
-members’ copyrighted sound recordings and music videos, as noted in Exhibit A hereto. For
-example, as shown on Exhibit A, the source code expressly suggests its use to copy and/or
-distribute the following copyrighted works owned by our member companies:
+... What's a "streisand effect"?
```

Comment:
Hahaha! Well put!

Reply:
What if it's intentional? Maybe the RIAA loved youtube-dl and want everyone to find it via reverse psychology! :o

Reply:
Well they definitely got that Streisand effect going good now hahahahah